modules/polaris/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Fixed
```

Comment:
This is a backport/merging to a release branch, so you'll be releasing the monorepo libraries immediately. As such you want to include the date and version here instead of putting it in `versions.txt`.